.github/lineage.yml

```
@@ -0,0 +1,5 @@
+---
+lineage:
+  skeleton:
+    remote-url: https://github.com/cisagov/skeleton-generic.git
```

Shouldn't this role descend from `skeleton-ansible-role`?
```suggestion
    remote-url: https://github.com/cisagov/skeleton-ansible-role.git
```

---

Great catch!